packages/boost.rb

```diff
@@ -3,15 +3,17 @@
 class Boost < Package
   description 'Boost provides free peer-reviewed portable C++ source libraries.'
   homepage 'http://www.boost.org/'
-  version '1.59.0'
-  source_url 'http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.gz'
-  source_sha256 '47f11c8844e579d02691a607fbd32540104a9ac7a2534a8ddaef50daf502baac'
+  version '1.64.0'
+  source_url 'https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2'
+  source_sha256 '7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332'
 
   def self.build
-    system './bootstrap --prefix=/usr/local'
+    system './bootstrap.sh'
   end
 
   def self.install
-    system './b2 install'
+    system "./b2 -a install"
+    system "mkdir -p #{CREW_DEST_DIR}/usr/local/lib"
+    system "cp /usr/local/lib/libboost_* #{CREW_DEST_DIR}/usr/local/lib"
```

Comment:
I installed boost with above option into ChromeOS armv7l and x86_64.  I also installed into Chromium OS x86_64 and i686.  All of them are succeed.  I guess you have a problem on your environment like other programs which boost uses like python or something.  Can you check the result of bootstrap.sh?  Mine shows below.
```
Bootstrapping is done. To build, run:

    ./b2
    
To adjust configuration, edit 'project-config.jam'.
Further information:

   - Command line help:
     ./b2 --help
     
   - Getting started guide: 
     http://www.boost.org/more/getting_started/unix-variants.html
     
   - Boost.Build documentation:
     http://www.boost.org/build/doc/html/index.html

Preconfiguring package...
Performing configuration checks

    - 32-bit                   : yes
    - arm                      : yes
    - symlinks supported       : yes
    - C++11 mutex              : no
    - lockfree boost::atomic_flag : yes
    - Boost.Config Feature Check: cxx11_auto_declarations : no
    - Boost.Config Feature Check: cxx11_constexpr : no
    - Boost.Config Feature Check: cxx11_defaulted_functions : no
    - Boost.Config Feature Check: cxx11_final : yes
    - Boost.Config Feature Check: cxx11_hdr_mutex : no
    - Boost.Config Feature Check: cxx11_hdr_tuple : no
    - Boost.Config Feature Check: cxx11_lambdas : no
    - Boost.Config Feature Check: cxx11_noexcept : no
    - Boost.Config Feature Check: cxx11_nullptr : no
    - Boost.Config Feature Check: cxx11_rvalue_references : no
    - Boost.Config Feature Check: cxx11_template_aliases : no
    - Boost.Config Feature Check: cxx11_thread_local : no
    - Boost.Config Feature Check: cxx11_variadic_templates : yes
    - has_icu builds           : no
warning: Graph library does not contain MPI-based parallel components.
note: to enable them, add "using mpi ;" to your user-config.jam
    - zlib                     : yes
    - bzip2                    : yes
    - iconv (libc)             : yes
    - icu                      : no
    - icu (lib64)              : no
    - native-atomic-int32-supported : yes
    - native-syslog-supported  : yes
    - pthread-supports-robust-mutexes : yes
    - compiler-supports-visibility : yes
    - gcc visibility           : yes
    - long double support      : yes
warning: skipping optional Message Passing Interface (MPI) library.
note: to enable MPI support, add "using mpi ;" to user-config.jam.
note: to suppress this message, pass "--without-mpi" to bjam.
note: otherwise, you can safely ignore this message.
    - zlib                     : yes
    - bzip2                    : yes

Component configuration:

    - atomic                   : building
    - chrono                   : building
    - container                : building
    - context                  : building
    - coroutine                : building
    - coroutine2               : building
    - date_time                : building
    - exception                : building
    - fiber                    : building
    - filesystem               : building
    - graph                    : building
    - graph_parallel           : building
    - iostreams                : building
    - locale                   : building
    - log                      : building
    - math                     : building
    - metaparse                : building
    - mpi                      : building
    - program_options          : building
    - python                   : building
    - random                   : building
    - regex                    : building
    - serialization            : building
    - signals                  : building
    - system                   : building
    - test                     : building
    - thread                   : building
    - timer                    : building
    - type_erasure             : building
    - wave                     : building
```